nep-Y.mediawiki

```diff
@@ -0,0 +1,80 @@
+
+Moreover, it is suggested that number of total consensus nodes is a variable not affected by votes in a ballot.   
+
+
+==Motivation==
+
+By allowing one NEO to vote for multiple consensus nodes, it opens for attacks by collecting a large amount of NEO, then nominating a large amount of consensus nodes and have them all vote for each other to secure >2/3 of bookkeeper nodes (see simulation results). While it would be expensive for one individual with malicious intentions to perform such an attack, there are economic incentives for consortiums to form and control the consensus nodes (to receive transaction fees). When NEO Council decentralizes the consensus nodes, a minority consortium will be able to form by social agreement, for example a private chat group, and agree to all vote for each other. Since number of total consensus nodes is decided by an algorithm based on number of votes in each ballot, the optimal game strategy for a consortium with economic gain as motive would be to spread out on as many candidates as possible, and then have all candidates vote for each other. This would allow the consortium with economic motive to increase the weighted average of votes in ballots, resulting in a higher number of total consensus nodes. Since a vote for multiple candidates is counted as a full vote to each candidate, this consortium will all vote for each other without any penalty or risk involved.  
+
+Consensus nodes are assigned in order by total number of votes. However, since the minority consortium affected the weighted average of votes in each ballot, the total number of consensus nodes may be vastly higher than the majority NEO holders expected. Using this strategy, the minority consortium can gain the majority of consensus nodes and thus also control the network.  
+
+The added complexity with this voting system is that the game strategy for a single individual who desire to become a consensus node is aligned with the game strategy for a minority consortium that desires to gain majority control. The single individual will have a larger chance of “making it in” as a consensus node if he splits his votes across many random candidates, increasing the weighted average and result in more consensus nodes. This will benefit the minority consortium mentioned above.  
+
+
+==Specification==
+
+1. Limit 1 NEO to 1 vote: voting for n candidates give each candidate 1/n votes.  
+
+2. Remove calculation of number of total consensus nodes as an algorithm of weighted average of votes in each ballot.  
+
+
+==Rationale for specification 1==
+
+By limiting 1 NEO to 1 vote, the minor consortium will need to sacrifice voting power if they want to get more candidates. This has three benefits:  
+
+1. The optimal strategy for the single individual that desires to become a consensus node will now change; his best chance is to vote only for himself, which in turn will be a disadvantage for consortiums (see point 2).  
```

Comment:
Maybe we can be more gender aware and inclusive using _"her"_ instead of _"him_"? :)

Reply:
him -> their
himself -> themself

@lerider